src/libraries/Microsoft.Extensions.Http/src/Logging/LoggingScopeHttpMessageHandler.cs

```diff
@@ -59,16 +59,25 @@ async Task<HttpResponseMessage> Core(HttpRequestMessage request, bool useAsync,
                 using (_logger.BeginRequestPipelineScope(request, out string? formattedUri))
                 {
                     _logger.LogRequestPipelineStart(request, formattedUri, shouldRedactHeaderValue);
-                    HttpResponseMessage response = useAsync
+
+                    try
+                    {
+                        HttpResponseMessage response = useAsync
```

Comment:
Please fix the formatting in the block (LoggingHttpMessageHandler above seems to be correct)